modules/loadbalancer/templates/nginx_balance_no_ssl.conf.erb

```diff
@@ -72,7 +72,13 @@ server {
     allow 10.0.0.0/8;
     allow 127.0.0.1;
     deny all;
```

Comment:
I will put deny all statement in line 74 and 80 after line 81. Should be there in all cases